modules/sno-du-applying-the-extra-installation-manifests.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: CONCEPT
```

Comment:
This reads more like a procedure to me, but I am guessing by the use of `<ul>` rather than `<ol>` here that the order is not important? Still think the metadata should be changed to PROCEDURE though.

Reply:
changed.